torch_xla/experimental/sparse/_coo_ops.py

```diff
@@ -0,0 +1,327 @@
+from contextlib import contextmanager
+from typing import Tuple
+import torch
+from torch.autograd import Function
+
+aten = torch.ops.aten
+
+_COO_DISPATCH_TABLE = {}
+_COO_FUNCTION_TABLE = {}
+
+from .coo import SparseCOOTensor
+
+
+def _register_dispatch_func(op):
+
+  def wrapper(f):
+    _COO_DISPATCH_TABLE[op] = f
+    return f
+
+  return wrapper
+
+
+def _register_function_func(op):
```

Comment:
Need to remove this, and the `__torch_function__` registrations. They are not a solution to the problem. Things are far more broken if we have a `__torch_function__` override.